CMakeLists.txt

```diff
@@ -47,8 +51,10 @@ option(SYSTEM_GLEW "Use system-installed GLEW instead of vendored one" OFF)
 
 if(SYSTEM_GLEW)
    find_package(PkgConfig REQUIRED)
+   find_package(OpenGL REQUIRED)
```

Comment:
Might be needed in the `SYSTEM_GLFW` as well, not tested. Worked without on my machine, CI + Homebrew complained about missing OpenGL symbols.